src/main/java/no/fdk/fdk_service_harvester/Application.java

```diff
@@ -0,0 +1,13 @@
+package no.fdk.fdk_service_harvester;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class Application {
```

Comment:
Er det mulig å få denne klassen over til Kotlin?

Reply:
Har møtt på bugs som kommer av pom o.l. prøver å finne main() før det har blitt produsert java fra kotlin-koden. Men det er jo over 1 år siden sist jeg prøvde, sikkert fiksa nå.

Så ja, sannsynligvis. Men jeg ser ikke helt hva vi vinner på det utenom et større potensial for mystiske bugs...